Overlay to sop

ejaronne opened the pull request:
Aligns the spreadsheet and overlay.rb to the latest InSpec profile lifecycle SOP located at:
https://cmsgov.github.io/saf/CMS_InSpec_Profile_Lifecycle_SOP_v1.0_20190702.pdf

ejaronne committed:
Update cms-ars3.1-mod-overlay.rb

ejaronne committed:
Update cms-ars3.1-mod-overlay.rb

ejaronne committed:
updated to align to latest lifecycle SOP

ejaronne committed:
restored final end statement

ejaronne committed:
corrected V-71855

to note that the SA-7  security control (from v1r4) is not in CMS ARS 3.1

ejaronne committed:
Add files via upload

aaronlippold reviewed (changes requested):
This control is ‘Not applicable’ since the related security control is not included in CMS ARS 3.1'

This wording seems a little more concise doesn’t it??

ejaronne committed:
Concise "This is Not Applicable" text

Changed 
"Not applicable for this CMS ARS 3.1 overlay,"
to
"This is Not Applicable"
